OneSignalSDK/unittest/src/test/java/com/test/onesignal/MainOneSignalClassRunner.java

```diff
@@ -1748,6 +1749,27 @@ public void shouldLogoutOfEmail() throws Exception {
       assertEquals("b2f7f966-d8cc-11e4-bed1-df8f05be55ba", logoutEmailPost.payload.get("app_id"));
    }
 
+   @Test
+   public void logoutEmailShouldNotSendEmailPlayersRequest() throws Exception {
+      // 1. Init OneSignal and set email
+      OneSignalInit();
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.